gguf-py/gguf/gguf_reader.py

```diff
@@ -251,6 +253,7 @@ def _build_tensors(self, start_offs: int, fields: list[ReaderField]) -> None:
             tensor_names.add(tensor_name)
             ggml_type = GGMLQuantizationType(raw_dtype[0])
             n_elems = int(np.prod(dims))
+            np_dims = tuple(reversed(dims.tolist()))
```

Comment:
`.tolist()` is necessary here to avoid an error when reshaping afterwards, something about the shape being of type `np.float64` for some reason.